GVFS/GVFS.FunctionalTests/Tools/GVFSServiceProcess.cs

```diff
@@ -3,15 +3,28 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Runtime.InteropServices;
 using System.ServiceProcess;
 using System.Threading;
 
 namespace GVFS.FunctionalTests.Tools
 {
     public static class GVFSServiceProcess
     {
-        public const string TestServiceName = "Test.GVFS.Service";
-        private const string ServiceNameArgument = "--servicename=" + TestServiceName;
+        private static readonly string ServiceNameArgument = "--servicename=" + TestServiceName;
+
+        public static string TestServiceName
+        {
+            get
+            {
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                {
+                    return "GVFS.Service";
+                }
+
+                return "Test.GVFS.Service";

```

Comment:
Why can't `TestServiceName` be "Test.GVFS.Service" on Mac?

If this does need to stay this way can you add a comment to the code?

Reply:
For testing we are Not launching the `Service` with custom command line args (that specify custom names for `Service` IPC pipe). So the `Service` registers the NamedPipe as `GVFS.Service*` and that is what is used by the `gvfs mount/unmount` and other commands to connect with the `Service` during testing.

Added a comment in `GVFSServiceProcess.cs`